website_event_ticket_limit/models/event_ticket.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2023 Le Filament (https://le-filament.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+from odoo import fields, models
+
+
+class EventTicket(models.Model):
+    _inherit = "event.event.ticket"
+
+    max_ticket = fields.Integer(string="Max by order", default=0)
```

Comment:
Why having a separate field instead of reusing the existing one?

Reply:
By existing one, I suppose you mean seats_max from event.type.ticket ?
It has not the same purpose : seats_max is used to limit the max number of this kind of tickets you can sell in total, when the new field limits the number of tickets you can sell per user.

For instance you may want to limit the max number of VIP tickets to 50 (because room does not allow for more, because you can only prepare food for up to 50 persons, or whatever reason), but each person can buy only 1 VIP tickets (and combined with other modules to limit duplicates or to authorize only logged in user you can enforce that no one can register twice or 2 tickets).

I understand from your question that the naming of the field was not clear enough so I renamed it `max_ticket_per_order`

Reply:
Yeah, I have mixed and although it's understandable to have a different one. The new name is better (although let's be consistent in the label - comments inline -)